src/components/common/Table/TableToolbar/TableToolbar.tsx

```diff
@@ -0,0 +1,89 @@
+import { createStyles, IconButton, makeStyles, Theme, Toolbar, Tooltip } from "@material-ui/core";
+import AddIcon from '@material-ui/icons/Add';
+import EditIcon from '@material-ui/icons/Edit';
+import { FunctionComponent, MouseEventHandler } from "react";
+import { TableInstance } from "react-table";
+import ColumnHidePage from "../ColumnHidePage/ColumnHidePage";
+import TableFilters from "../TableFilters/TableFilters";
+import GlobalFilter from "../filters/GlobalFilter";
+
+const useStyles = makeStyles((theme: Theme) => {
+  return createStyles({
+    toolbar: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      border: '1px solid rgba(224, 224, 224, 1)',
+      borderTopLeftRadius: 10,
+      borderTopRightRadius: 10,
+      borderBottom: 'none',
+      background: "#fff",
+      position: 'sticky',
+      top: 0
+    },
+    left_icons: {
+      display: 'flex',
+      alignItems: 'center'
+    },
+    right_icons: {
+      display: 'flex',
+      alignItems: 'center'
+    }
+  })
+})
+
+type TableToolbarProps = {
+  instance: TableInstance,
+  onAdd?: (instance: TableInstance) => void
+  onEdit?: (instance: TableInstance) => void
+}
+
+const TableToolbar: FunctionComponent<TableToolbarProps> = ({ instance, onAdd, onEdit }) => {
+  const classes = useStyles({});
+  const {state} = instance;
+
+  return (
+    <Toolbar className={classes.toolbar}>
+      <div className={classes.left_icons}>
+        {onAdd && 
+          <Tooltip title="Add new event">
+            <div>
+              <IconButton 
+                onClick={() => onAdd(instance)} 
+                aria-label="Add new event"
+                disabled={!(Object.keys(state.selectedRowIds).length === 0)}
```

Comment:
```suggestion
                disabled={Object.keys(state.selectedRowIds).length !== 0}
```

Reply:
May be will be enough
`Object.keys(state.selectedRowIds).length`